tests/integration/changed-yum-conf/test_patch_yum_conf.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
+
+from envparse import env
+
+
+@pytest.mark.good_tests
+def test_yum_patch(convert2rhel, shell):
+    with convert2rhel(
```

Comment:
Add here:
shell("echo '#random text' >> /etc/yum.conf") and you're done. No need additional ansible playbook

Reply:
And actually you can run just shell("touch /etc/yum.conf"). Touching it is enough. A touched file (modified date gets updated) is considered changed by the rpm tooling.

Reply:
I'm taking the _touch_ idea back. We are [checking the MD5 checksum](https://github.com/oamg/convert2rhel/pull/189/files#diff-f06a581f26424ede8990e290a85424d0d56edf4a5c5a18c97c293ee9dd72364bR106), and touching does not change that.

Reply:
It brings me to the question - should we be checking the MD5 checksum? I have a hunch that whatever change happens to the file, yum won't replace it during reinstallation. If that's the case, we'd need to be checking only whether the file appears in the `rpm -V` output. I need to verify that.

Reply:
I checked the yum spec on RHEL 7 and the /etc/yum.conf is `%config(noreplace)`.
My testing shows, that when I change the mode (chmod) or owner or timestamp, the config gets replaced.
So, no change needed, you're doing it right ;)